tests/test_routes.py

```diff
@@ -0,0 +1,53 @@
+def test_get_one_planet(client, two_saved_planets):
+    # Act
+    response = client.get("/planets/1")
+    response_body = response.get_json()
+
+    # Assert
+    assert response.status_code == 200
+    assert response_body == {
+        "id": 1,
+        "name": "Mars",
+        "description": "Smallest planet"
+    }
+
+
+def test_get_one_planet_no_data_returns_404(client):
+    # Act
+    response = client.get("/planets/1")
+    response_body = response.get_json()
+
+    # Assert
+    assert response.status_code == 404
+
+
+def test_get_all_planets(client, two_saved_planets):
+    # Act
+    response = client.get("/planets")
+    response_body = response.get_json()
+
+    # Assert
+    assert response.status_code == 200
+    assert response_body == [{
+        "id": 1,
+        "name": "Mars",
+        "description": "Smallest planet"
+        },
+        {
+        "id": 2,
+        "name": "Jupiter",
+        "description": "Largest planet"
+        }]
```

Comment:
If the fixture two_saved_planets returned two planet objects, you'd be able to compare what's in the response body against what was used to create records in the database (as we do in Flasky)